submissions/AaLin-Git/html-css-popup/style.css

```diff
@@ -0,0 +1,270 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*,
+*:before,
+*:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+:focus,
+:active {
+  outline: none;
+}
+
+.container,
+.footer,
+.header {
+  display: block;
+}
+
+html,
+.body {
+  font-family: Verdana, Tahoma, sans-serif;
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 13px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+.input::-ms-clear {
+  display: none;
+}
+
+.menuPopup,
+.menuPopupSub {
+  display: none;
+}
+
+.wrapper,
+.main,
+.footer,
+.container,
+.content,
+.menuPopupItem {
+  display: flex;
+}
+
+.contentItem,
+.labelIcon {
+  display: inline-block;
+}
+
+.hiddenNavbar,
+.content {
+  position: relative;
+}
+
+.input,
+.menuPopup,
+.menuPopupSub,
+.menuInputSub {
+  position: absolute;
+}
+
+.wrapper,
+.main,
+.menuPopupItem {
+  flex-direction: column;
+}
+
+.main,
+.footer,
+.content,
+.menuPopupItem {
+  align-items: center;
+}
+
+.main,
+.container {
+  justify-content: flex-end;
+}
+
+.footer,
+.menuPopupItem,
+.more {
+  justify-content: center;
+}
+
+.header,
+.footer {
+  height: 100px;
+  background-color: #fffce8;
+}
+
+.content,
+.contentItem,
+.labelIcon {
+  padding: 5px;
+}
+
+.menuPopupItem {
+  margin: 5px;
+  padding: 5px 0;
+}
+
+.footer,
+.textItem,
+.more,
+.contentItem {
+  color: #4462ab;
+}
+
+.contentItem,
+.menuPopupItem,
+.more {
+  text-decoration: none;
+}
+
+.contentItem,
+.labelIcon,
+.menuPopupItem {
+  border-radius: 4px;
+}
+
+.menuPopup,
+.menuPopupSub {
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.wrapper {
+  min-height: 100%;
+  overflow: hidden;
+}
+
+.main {
+  flex: 1 1 auto;
+  height: 1fr;
+}
+
+.backgroundImage {
+  height: 30%;
+}
+
+.content {
+  margin: 15px 30px;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 50%;
+  list-style: none;
+}
+
+.hiddenNavbar section,
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Also, avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
Thanks, I removed section. Should I  also remove * and htms tags from the reset CSS?

Reply:
No, in reset.css you can use HTML tags because you set basic styles in that file.